### PR TITLE
Refresh tab-icons when changing item's title

### PR DESCRIPTION
**Pull request by Alhadis**
A blank editor doesn't update its tab's icon after it's been saved to disk. Meaning that the file-icon service doesn't have a chance to show anything in the tab until it's been closed and reopened.

### CODE REVIEW

File: spec/tabs-spec.coffee [hunk 1]
```diff
@@ -1351,3 +1353,72 @@ describe "TabBarView", ->
 
         runs ->
           expect($(tabBar).find('.tab:eq(1) .title')).toHaveClass "status-modified"
+
+describe "File-icons service", ->
```
Comment (50Wliu) [inline spec/tabs-spec.coffee]: In other packages this has been added as a standalone spec file `file-icons-spec.coffee`.

Comment (50Wliu) [inline spec/tabs-spec.coffee]: He knows better than I do :)

Comment (Alhadis) [inline spec/tabs-spec.coffee]: Hence why I'm relying on precedent scenarios. =)

I'm hoping #966 gets reviewed sooner, though. The hacks I'm writing to monkey-patch the tree-view's file-moving routine is nauseating.

**Comment by Alhadis** [comment]
Fixed in #402.

**Pull request closed without merging**